### PR TITLE
Feature/asm 231 241

### PR DESCRIPTION
**Pull request by aaronmallen**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -5,6 +5,10 @@ class CommentsController < ApplicationController
   def index
     @comments = @commentable.comments
   end
+
```
Comment (wasi) [inline app/controllers/comments_controller.rb]: have a look at the basic controller spec scaffolding:
https://github.com/rspec/rspec-rails/blob/master/lib/generators/rspec/scaffold/templates/controller_spec.rb

File: spec/models/comment_spec.rb [hunk 2]
```diff
@@ -21,4 +21,9 @@
     @comment.commenter = nil
     expect !@comment.valid?
   end
+
```
Comment (wasi) [inline spec/models/comment_spec.rb]: you could test for validations aswell: https://github.com/asm-products/mana-rails/blob/develop/spec/models/client_spec.rb#L6
https://github.com/thoughtbot/shoulda-matchers

File: app/views/comments/_comments.html.slim [hunk 3]
```diff
@@ -1,7 +1,8 @@
 - @comments.each do |comment|
-  .panel.panel-default
+  .panel.panel-default id="#{comment.slug}"
     .pull-right.m1
-      = link_to raw('<i class="fa fa-trash"></i>'), [@client, comment], method: :delete, data: { confirm: 'Are you sure?' },class: "btn btn-default btn-sm btn--white mr1", title: "Delete Note"
+			= link_to raw('<i class="fa fa-trash"></i>'), ([@commentable, comment]), method: :delete, data: { confirm: 'Are you sure?' },class: "btn btn-default btn-sm btn--white mr1", title: "Delete Note"
```
Comment (wasi) [inline app/views/comments/_comments.html.slim]: you could test that with request specs aswell

**Comment by aaronmallen** [comment]
@wasi submitted requested specs, with the exception of the client_request_spec.  I left what I did commented out so that I could get your help on it later.

**Pull request merged**